### DISCUSSION TITLE
Two links for the same web

### DISCUSSION THREAD

**Discussion by borekon** [YOURLS/YOURLS#3096] [Q&A] [answered]
Hi,
I was looking in the issues and the discussion, but no answer found.
I want to create two redirections for the same website (for statistics purposes i.e. where are the visitors coming from) but yourls throws an error that says the destination url already exists.
Is there any way tho skip that restriction?

**Comment by ozh** [comment, marked as answer]
This is a config setting that is detailed in the readme.
